tensorflow/core/common_runtime/eager/mkl_eager_op_rewrite.cc

```diff
@@ -75,12 +76,16 @@ REGISTER_REWRITE(EagerOpRewriteRegistry::PRE_EXECUTION, MklEagerOpRewrite);
 // Constructor
 MklEagerOpRewrite::MklEagerOpRewrite(string name, string file, string line)
     : EagerOpRewrite(name, file, line) {
+  mkl_eager_ops_.push_back({"BatchMatMulV2", AlwaysRewrite,
+                            CreateGenericMklOp});  // No need to check for V1 as
+                                                   // it has been obsoleted
```

Comment:
What if people decide to run an old graph with eager mode?

Reply:
@penpornk I can add it also, but I thought since python is code is calling V2 now we won't see V1 here.

Reply:
It can still be called in Python through `gen_math_ops.batch_mat_mul`, but up to you. :)

Reply:
Thanks @penpornk  will add it also.